src/ext/debug_backtrace.c

```diff
@@ -0,0 +1,39 @@
+#include <execinfo.h>
```

Comment:
We'll need to add `debug_backtrace.c`, `debug_backtrace.h` and the two new test files to `package.xml` for PECL. :)